WordPress/gutenberg
# Storybook: Add stories for BlockAlignmentControl and BlockAlignmentToolbar components

Part of https://github.com/WordPress/gutenberg/issues/67165

## What?

This PR will add stories for `BlockAlignmentControl` and `BlockAlignmentToolbar` components in the Storybook

## Why?

Most of Block Editor components don't have stories.
  
## Testing Instructions




1. Run `npm run storybook:dev` 
2. Open the storybook on http://localhost:50240/
3. Check the `BlockAlignmentControl` and `BlockAlignmentControl` stories.

## Screenshots or screencast 
 
 
 
[BlockEditor-BlockAlignmentControl---Docs-⋅-Storybook.webm](https://github.com/user-attachments/assets/f37d182a-beaa-46b3-a5cf-33cc6c828b3c)
